pkg/constants/constants.go

```diff
@@ -223,6 +228,10 @@ func CanaryServiceName(name string, component InferenceServiceComponent) string
 	return name + "-" + component.String() + "-" + InferenceServiceCanary
 }
 
+func MultiModelConfigMapName(inferenceserviceName string, shardId int) string {
```

Comment:
Consider calling this ModelConfigMap instead of specifying "multi". IIUC, this will be the new path for all model serving types.

Reply:
Alright, refactored to 
```
func ModelConfigMapName(inferenceserviceName string, shardId int) string {
	return fmt.Sprintf("modelconfig-%s-%d", inferenceserviceName, shardId)
}
```